benchmarking.py

```diff
@@ -0,0 +1,108 @@
+import os
+import torch
+from hiera.hiera import Hiera
+from hiera.benchmarking import benchmark
+
+def main():
+    # Set environment variables
+    os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+    os.environ["WORLD_SIZE"] = "1"
+
+    # Model configuration
+    MODEL_BACKBONE = "hiera-2d"  # Options: 'hiera-2d' or 'hiera-3d'
+    BATCH_SIZE = 64
+    SCREEN_CHUNK_SIZE = 30
+    SCREEN_SAMPLING_RATE = 30
+    RESPONSE_CHUNK_SIZE = 8
+    RESPONSE_SAMPLING_RATE = 8
+    BEHAVIOR_AS_CHANNELS = True
+    REPLACE_NANS_WITH_MEANS = True
+    DIM_HEAD = 64
+    NUM_HEADS = 2
+    DROP_PATH_RATE = 0
+    MLP_RATIO = 4
```

Comment:
for future - by pep8 caps is used for global consts and they are typically defined right under imports and not in the functions:)

Reply:
can we fix this + move the function itself into `hiera.benchmarking` like `def main() -> def benchmarkark_flash_attention()` (or move the file to `hiera` folder but the first option is more prefered)

I am sorry for being annoying but as many people might use it, better to do it clean from the beginning